packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -93,6 +93,7 @@ function useContext<T>(
   context: ReactContext<T>,
   observedBits: void | number | boolean,
 ): T {
+  nextHook();
```

Comment:
not sure if relevant, but we add the workinprogresshook to the list only when __DEV__. so `hook.next` behaves differently in prod vs dev https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberHooks.js#L527-L529

Reply:
Good catch. My test had a blindspot here since the null hook would just re-initialize the state.

Reply:
Hmm... 🤔 I think this might be trickier than a simple `__DEV__` check can account for. It's not a matter of whether `react-debug-tools` is itself in production or development mode, but whether the renderer it's interacting with is.